src/core/raster/qgsrasterlayertemporalproperties.h

```diff
@@ -23,6 +23,7 @@
 #include "qgis_sip.h"
 #include "qgsrange.h"
 #include "qgsmaplayertemporalproperties.h"
+#include "qgsrasterdataprovider.h"
```

Comment:
Some from https://github.com/qgis/QGIS/commit/313b000147d28bb7eea788774890af62c1c60b9f were reverted here, can you please fix?